cypress/integration/myLists/myFavorites.spec.js

```diff
@@ -0,0 +1,29 @@
+import { MY_LIST_ROUTE } from '../../../src/config/routes';
+import { PUBLISHED_ITEMS } from '../../fixtures/items';
+import {
+  MY_FAVORITES_COLLECTIONS_GRID,
+} from '../../support/selectors';
+import { ITEM_PUBLISHED_TAG } from '../../fixtures/itemTags';
+import { buildMemberWithFavorites, FAVORITE_ITEMS } from '../../fixtures/members';
+
+describe('My Favorite Items', () => {
+  describe('Current user', () => {
+    // check if title and headings are displayed correctly
+    it('display favorite items', () => {
+      cy.setUpApi({currentMember: buildMemberWithFavorites(FAVORITE_ITEMS), items: PUBLISHED_ITEMS});
```

Comment:
Instead of creating by hand `FAVORITE_ITEMS`, why not pick them from `PUBLISHED_ITEMS` before `setUpApi`?